src/main/java/frc/robot/subsystems/Lift.java

```diff
@@ -1,36 +1,128 @@
 package frc.robot.subsystems;
 
-import com.chopshop166.chopshoplib.outputs.SendableSpeedController;
+import java.util.function.BooleanSupplier;
+import java.util.function.DoubleSupplier;
+
+import com.chopshop166.chopshoplib.outputs.ISolenoid;
+import com.chopshop166.chopshoplib.outputs.PIDSpeedController;
+import com.chopshop166.chopshoplib.sensors.IEncoder;
+import com.chopshop166.chopshoplib.sensors.InvertDigitalInput;
+import com.google.common.collect.Range;
+
+import org.checkerframework.checker.units.qual.Speed;
 
 import edu.wpi.first.wpilibj2.command.CommandBase;
-import edu.wpi.first.wpilibj2.command.StartEndCommand;
+import edu.wpi.first.wpilibj2.command.FunctionalCommand;
+import edu.wpi.first.wpilibj2.command.InstantCommand;
 import edu.wpi.first.wpilibj2.command.SubsystemBase;
 import frc.robot.maps.RobotMap;
 
-public class Lift extends SubsystemBase {
+/*
+ * SUB REVIEW Lift Subsystem What Does It Do? 1) Raises and lowers lift Manually
+ * using two different moters in sync during the last 30 seconds of the game 2)
+ * Uses ratchet mechanism to break
+ * 
+ * What Modes Does It Have? 1) Up (Extend) 2) Down (Retract) 3) Brake (Ratchet)
+ * 4) Hook onto Bar using d-clip 5) Go to Middle Bar Position 6) Go to Top Bar
+ * Position
+ * 
+ * What Interactions Does It Have With Other Subsystems? 1) None
+ * 
+ * How Is It Triggered? 1) User Input
+ * 
+ * Does It Store Any State? 1) isBraked 2) Limit switch values 3) Encoder values
+ * 
+ * Sensors? 1) Encoders 2) Limit Switches
+ */
 
-    private SendableSpeedController elevatorMotor;
+public class Lift extends SubsystemBase {
 
+    private PIDSpeedController elevatorMotor;
+    private ISolenoid elevatorBrake;
+    private IEncoder liftEncoder;
+    private BooleanSupplier upperLimitSwitch;
+    private BooleanSupplier lowerLimitSwitch;
     private static final double elevatorMotorSpeed = 1;
+    private static final double TOLERANCE_RANGE = 5;
```

Comment:
What unit is this in? Please put the unit in the name (eg `TOLERANCE_RANGE_INCHES`).

Also, assuming inches 5 is way to large, this should probably be 0.5 or so.